src/main/java/nl/tudelft/semgroup4/resources/Resources.java

```diff
@@ -123,101 +38,169 @@ public static void init() throws SlickException {
-        bubbleImage2 = new Image("src/main/resources/img/yball2.png");
-        bubbleImage3 = new Image("src/main/resources/img/yball3.png");
-        bubbleImage4 = new Image("src/main/resources/img/rball4.png");
-        bubbleImage5 = new Image("src/main/resources/img/rball5.png");
-        bubbleImage6 = new Image("src/main/resources/img/rball6.png");
-
-        pickupWeaponRegular = new Image("src/main/resources/img/pickup_regular_weapon.png");
-        pickupWeaponDouble = new Image("src/main/resources/img/pickup_weapon_double.png");
-        pickupWeaponSticky = new Image("src/main/resources/img/pickup_sticky.png");
-        pickupWeaponFlowers = new Image("src/main/resources/img/pickup_flowers.png");
-        pickupPowerShield = new Image("src/main/resources/img/pickup_shield.png");
-        pickupPowerInvincible = new Image("src/main/resources/img/pickup_invincible.png");
-        pickupPowerMoney = new Image("src/main/resources/img/pickup_money.png");
-        pickupPowerPoints = new Image("src/main/resources/img/pickup_points.png");
-        pickupPowerSpeedup = new Image("src/main/resources/img/pickup_speed.png");
-        pickupUtilitySplit = new Image("src/main/resources/img/pickup_split.png");
-        pickupUtilityFreeze = new Image("src/main/resources/img/pickup_freeze.png");
-        pickupUtilitySlow = new Image("src/main/resources/img/pickup_slow_down.png");
-        pickupUtilityLevelwon = new Image("src/main/resources/img/pickup_level_won.png");
-        pickupUtilityTime = new Image("src/main/resources/img/pickup_time.png");
-        pickupUtilityLife = new Image("src/main/resources/img/pickup_life.png");
+                        Image image = new Image(fileString);
         
+                        // Add the image belonging to this path to the hashmap. The
+                        // hashmap belongs to one of the folders in animation.
+                        animationImages.get(animationHash).put(fileString, image);
+    
+                    } catch (Exception e) {
+                        e.printStackTrace();
+                    }
+                }
+            });
 
-        powerInvincible = new Image("src/main/resources/img/powerup_invincible.png");
-        powerShield = new Image("src/main/resources/img/powerup_shield.png");
-        
-        Font font = new Font("Calibri", Font.BOLD, 60);
-        countdownFont =  new TrueTypeFont(font, true);
+        makeAnimations(animationImages);
+    }
+
+    /**
+     * Loops over each item of the hashmap. Each item consists of a hashmap, which contains images
+     * belonging to one animation. The key of the outer hashmap is used as key to store the
+     * animtion.
+     */
+    private static void makeAnimations(HashMap<String, HashMap<String, Image>> animationImages) {
+        // loop over all the items in the hashmap
+        for (String key : animationImages.keySet()) {
+            HashMap<String, Image> curAnimationImages = animationImages.get(key);
```

Comment:
The name is too long. Personally think that the name is OK, but maybe remove the 'cur' part (if it fixes the PMD warning), it is a variable scoped in a for loop, it obviously is about the current iteration of that loop.

< naming